podcasts/HomeGridListItem.swift

```diff
@@ -58,6 +58,8 @@ class HomeGridListItem: ListItem {
                 folder.name == otherFolder.name &&
                 folder.color == otherFolder.color &&
                 folder.syncModified == otherFolder.syncModified &&
+                folder.sortType == otherFolder.sortType &&
+                folder.sortOrder == otherFolder.sortOrder &&
```

Comment:
Why do we need this addition?

Reply:
From my understanding this is for the DifferenceKit algorithm that we are using to distinguish updates on the collection view to work correctly when there are changes on the folder sort order.